internal_transfers/actions/tests/database.spec.ts

```diff
@@ -25,16 +26,19 @@ const dbURL: string =
   process.env["DATABASE_URL"] ||
   "postgresql://postgres:postgres@localhost:5432/postgres";
 const db = getDB(dbURL);
+async function truncateTables() {
+  await db.query(sql`TRUNCATE TABLE settlements;`);
+  await db.query(sql`TRUNCATE TABLE internalized_imbalances;`);
+  await db.query(sql`TRUNCATE TABLE settlement_simulations;`);
+  await db.query(sql`TRUNCATE TABLE tx_receipts;`);
```

Comment:
Very nit: Can be wrapped in a for loop for each table.

Reply:
Fixed.

Reply:
Nope -- sorry not gonna do this now:

Feel free to try it out if you'd like

https://github.com/cowprotocol/solver-rewards/actions/runs/4946979712/jobs/8845749193